handbook/engineering/roles.md

```diff
@@ -26,30 +27,56 @@ Some teams with more experienced members may need less guidance in certain areas
 ### Responsibilities
 
 - Directly contribute through code, RFCs or PR reviews for a minimum of roughly 50% of their time.
-- Collaborate with [product](../product/index.md) to maintain a [prioritized roadmap](../../direction/index.md) for the work owned by their team as well as quarterly [OKRs](../../company/okrs/index.md)
-- Plan and commit to a set of work that will be accomplished by the team for each [monthly release](releases/index.md) and capture those commitments in a [tracking issue](tracking_issues.md) on GitHub.
+- Ensure that the team has a roadmap that documents how it is going to achieve its [OKRs](../../company/okrs/index.md).
+- Ensure the the team's [tracking issue](tracking_issues.md) captures the work that each teammate has committed to for each [monthly release](releases/index.md).
+- Post a weekly update by EOD Monday on the tracking issue that answers the following questions:
+  1. What work did the team do last week?
+    - This is a summary not details. The details should exist in each teammate's individual update and shouldn't be repeated.
+  1. How does that work connect back to the team’s OKRs and roadmap?
+  1. For each OKR and roadmap item, is the team on track? The answer should be of the form:
+    - We expect to reach this goal.
+    - We are at risk of not reaching this goal because A, B, C.
+      - What can we change to get this back on track?
+    - We are not going to reach this goal because A, B, C.
+      - What did we learn from this failure to achieve our goals?
+  1. What did the team work on last week that wasn’t originally planned? Why?
```

Comment:
How granular should this be? My observation is that unplanned work typically involves to resolving very specific issues. But it also seems hard to track at a granular level (e.g. someone may fix a small bug and not put it in their status update, or they spend a few hours hacking on something semi-relevant on the side). 

Can we add an example of what this would look like? Would help clarify, given we say "This is a summary not details. The details should exist in each teammate's individual update and shouldn't be repeated" above too.

Reply:
@attfarhan - what I would like from this item is the bigger things that prevented the team from getting their planned work done. Small tasks such as bug fixes or side efforts fit into the planned "slack time" that each engineer should have. Things like, this high priority customer issue was filed and I had to spend 2 days fixing it or there was an unexpected complexity to the feature I'm working on and I had to spend 3 days fixing this other code that impacted my feature. Similarly, understanding patterns within the team - the team all spent 1-2 days working on improving test coverage as a spike and this was not originally planned for this week.

Does that help?

Reply:
Makes sense 👍  thanks!

Reply:
I moved the docs to the tracking issue page and updated the wording. PTAL and suggest any edits that you think would make it more clear.